### DISCUSSION TITLE
Can I use govt I'd(aadhar card) for verification?

### DISCUSSION THREAD

**Discussion by Tech-with-anmol** [community/community#139578] [GitHub Education] [answered]
### Hi everyone,

I had applied for the student pack but got rejected due to 2 reasons:

1. I was not near the campus
2. My name written on receipt and fee card of school which i used as proof has bad handwriting so my last name is not visible clearly and written terribly. 

So, the thing is I can't do anything about it, so can I show aadhar card insated to verify status. Also what other thing I can do.

**Comment by VadymMakohon** [comment, marked as answer]
Hey, I totally get how frustrating that must be! It’s such a bummer that your GitHub Student Pack got rejected over something like handwriting and your distance from campus.  It happened to me twice.
Here is what i did when i was in your situation:
1. I personally do not see correlation with school wifi and your rejection, i applied from home myself.
2. Contacted the education center and asked for the fresh receipt where all the info is written clearly.
3. If they don’t accept your docs again, don’t give up. A lot of the individual tools and services in the GitHub Student Developer Pack offer their own student discounts or trials. So even without the pack, you might still be able to access some of the perks by going directly through the other companies.

Hope it will helps and if it does, would you be so kind to mark it as the answer?

Regards,

Vadym.

**Reply by Tech-with-anmol** [reply]
Thanks for replying, but my application got approved in second try. I did use smartphone that solved location problem and I used more documents as proof for their clarity.
